docs/cli.rst

```diff
@@ -0,0 +1,988 @@
+^^^^^^^^^^^^^^^^^^^^
+
+::
+
+    $ nextflow help [options]
+
+
+Extended description
+^^^^^^^^^^^^^^^^^^^^
+This command is equivalent to simply issuing ``nextflow`` at the command line - it prints out the overview of the CLI interface and enumerates the top-level *options* and *commands*.
+
+
+Options
+^^^^^^^^^^^^^^^^^^^^
+
++---------------------------+------------+--------------------------------------------------------------------------------+
+| Name, shorthand (if any)  | Default    | Description                                                                    | 
++===========================+============+================================================================================+
+| -help, -h                 |  false     | Print the command usage.                                                       |
++---------------------------+------------+--------------------------------------------------------------------------------+
+
+
+
+Examples
+^^^^^^^^^^^^^^^^^^^^
+
+
+--------------------
+info          
+--------------------
+
+
+Description
+^^^^^^^^^^^^^^^^^^^^
+Print project and system runtime information
+
+
+
+Usage
+^^^^^^^^^^^^^^^^^^^^
+
+::
+
+    $ nextflow info [options]
+
+
+
+Extended description
+^^^^^^^^^^^^^^^^^^^^
+
+The ``info`` command prints out the nextflow runtime information about the hardware as well as the software versions of the ``Nextflow version and build``, ``Operating System`` and ``Groovy and Java runtime``.
+
+Options
+^^^^^^^^^^^^^^^^^^^^
+
++---------------------------+------------+--------------------------------------------------------------------------------+
+| Name, shorthand (if any)  | Default    | Description                                                                    | 
++===========================+============+================================================================================+
+| -check-updates, -u        |  false     | Check for remote updates.                                                      |
++---------------------------+------------+--------------------------------------------------------------------------------+
+| -d                        |  false     | Show detailed information.                                                     |
++---------------------------+------------+--------------------------------------------------------------------------------+
+| -help, -h                 |  false     | Print the command usage.                                                       |
++---------------------------+------------+--------------------------------------------------------------------------------+
+| -o                        |  text      | Output format, either ``text``, ``json`` or ``yaml``.                          |
++---------------------------+------------+--------------------------------------------------------------------------------+
+
+
+
+Examples
+^^^^^^^^^^^^^^^^^^^^
+
+
+--------------------
+kuberun       
+--------------------
+
```

Comment:
@evanfloden I'd like to confirm, for this iteration, how extensive should be the `EXAMPLES` section for `kuberun` and `run`.

Here are the possible choices:


- Should we document everything extensively right now as the `CLI` doc would probably be referenced quite frequently?

- Should we focus only on the most frequent use-cases and add more examples as per user requests?

Reply:
I've opted for the most frequent ones and would add more in case there is user request or I get more ideas.